### PR TITLE
Mobile view

### PR DESCRIPTION
**Pull request by FaridaAdamuMagaji**
Discover your dream destination: my travel website eases travel planning.
This Pull request introduces Redesigned destination pages, optimized mobile responsiveness, improved performance and accessibility.
Asset organization: 
-Created a dedicated images folder to store and manage project images to improve code organization and enhance visual appeal. 
Testing:
-Verified styling updates using a browser.

Please review and provide feedback. 

Best regards, 
Farida Adamu Magaji.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        </h4>
+        <br> 
+        <section class="fig">
+            <div class="location">
+                <img src="Location.svg" alt="Location icon">
+            </div>
+            <div class="place">
+                Jawa Tengah, Indonesia
+            </div>
+            <br>
```
Comment (Devsalmasaeed) [inline index.html]: For best practice avoid using br in your code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    <section class="pics">
+        <img src="Image (13).jpg" alt="Lake">
+    </section>
+    <br>
+    <h6 class="caption">
+        Sunter Lake
+    </h6>
+    <br>
+    <section class="fig">
+        <div class="location">
```
Comment (Devsalmasaeed) [inline index.html]: Avoid using too much br in your code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+            </div>
+            <div>
+                <img src="arrow-white.svg" alt="arrow icon">
+            </div>
+        </section>
+    </section>
+    <br>
+    <br>
+    <br>
+    <br>
```
Comment (Devsalmasaeed) [inline index.html]: Make sure your code is well indented

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]
Hi @FaridaAdamuMagaji ,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- Please create different folder for your images
- Make sure to used professional PR title and description
- For best practice avoid using too much break in your code
- Make sure your code is well indented
- Used the same font-family,font-weight as used in figma
- Make sure your design is the same as the figma design the following is the area you should focus on

Step 2 screenshot

<br/>


Step 1 screenshot

<br/>

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]
> Hi @FaridaAdamuMagaji ,
> 
> Good job so far! You still need to work on some issues to get your project approved, but you are almost there!
> 
> Remote Collaboration
> ### To Highlight 💯
> * **Clean code** 👍
> * **Good use of Gitflow** 👍
> * **Implemented some project requirements** 👍
> 
> ### Required Changes ♻️
> * Make sure to used professional PR title and description
> * Make sure your design is the same as the figma design the following is the area you should focus on
> 
> Step 2 screenshot
> 
> 
> Step 1 screenshot
> 
> 
> ### Optional Suggestions
> Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
> 
> Cheers and Happy coding! 👏👏👏
> 
> _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification._

**Comment by FaridaAdamuMagaji** [comment]
Good day ma, i have already made changes to the two pictures above in my previous submission. @Devsalmasaeed 
IMG_20241023_172007_732
IMG_20241023_172105_747

**Comment by Devsalmasaeed** [comment]
It's not reflecting on your website you met me @ Aisha Kwaku today 10am-3pm

**Comment by FaridaAdamuMagaji** [comment]
ok @Devsalmasaeed

**Comment by Xfarouq** [comment]
Hey @FaridaAdamuMagaji 
- make your video section i.e. the div which you put the video image and also the video image to all be 100% to avoid that white gap by the right of the website
- Also make sure all your images have title names for best practices
- Always make sure to name your images folder as assets not pictures for best practices
- Try to centrally align the second section please
Remember this is your last trial so try to rectify all corrections please

**Review by Devsalmasaeed — APPROVED** [review]
Hi @FaridaAdamuMagaji ,

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Pull request merged**